nupic/research/monitor_mixin/temporal_memory_monitor_mixin.py

```diff
@@ -383,10 +389,56 @@ def mmClearHistory(self):
 
     self._mmTraces["predictedCells"] = IndicesTrace(self, "predicted cells")
     self._mmTraces["activeColumns"] = IndicesTrace(self, "active columns")
+    self._mmTraces["activeCells"] = IndicesTrace(self, "active cells")
     self._mmTraces["predictiveCells"] = IndicesTrace(self, "predictive cells")
+    self._mmTraces["correctlyPredictedCells"] = IndicesTrace(self, "correctly "
+                                                             "predicted cells")
     self._mmTraces["numSegments"] = CountsTrace(self, "# segments")
     self._mmTraces["numSynapses"] = CountsTrace(self, "# synapses")
     self._mmTraces["sequenceLabels"] = StringsTrace(self, "sequence labels")
     self._mmTraces["resets"] = BoolsTrace(self, "resets")
-
     self._mmTransitionTracesStale = True
+
+
+  def mmGetCellActivityPlot(self, title="", showReset=False,
```

Comment:
If this function is overriding a parent, it should have the same function signature as the parent. Otherwise, it should be a different function, and the parent should be treated as a helper function.